Formula/openimageio.rb

```diff
@@ -62,22 +81,6 @@ def install
       -DUSE_QT=OFF
     ]
 
-    # CMake picks up the system's python shared library, even if we have a brewed one.
-    python3 = python.opt_bin/"python3"
-    py3ver = Language::Python.major_minor_version(python3)
-    py3prefix = if OS.mac?
-      python.opt_frameworks/"Python.framework/Versions"/py3ver
-    else
-      python.opt_prefix
-    end
-
-    ENV["PYTHONPATH"] = prefix/Language::Python.site_packages(python3)
-
-    args << "-DPython_EXECUTABLE=#{python3}"
-    args << "-DPYTHON_LIBRARY=#{py3prefix}/lib/#{shared_library("libpython#{py3ver}")}"
-    args << "-DPYTHON_INCLUDE_DIR=#{py3prefix}/include/python#{py3ver}"
```

Comment:
No longer used probably after changes like https://github.com/OpenImageIO/oiio/pull/2705 and https://github.com/OpenImageIO/oiio/pull/2764
```
  Manually-specified variables were not used by the project:

    PYTHON_INCLUDE_DIR
    PYTHON_LIBRARY
    USE_FIELD3D
```

Also, Field3d support was deprecated and isn't used unless `-DENABLE_FIELD3D`